scripts/generate-google3-docs/resource-reference/overview.md

```diff
@@ -830,6 +830,10 @@ issues for {{product_name_short}}.
   <td><a href="/vpc/docs/">{{serverless_vpc_access_name}}</a></td>
   <td><a href="/config-connector/docs/reference/resource-docs/vpcaccess/vpcaccessconnector.md">VPCAccessConnector</a></td>
 </tr>
+<tr>
```

Comment:
Similarly, could you move this section up?

Reply:
Done